diff --git a/src/app/event.service.ts b/src/app/event.service.ts
index 52bfb60..09d0bc0 100644
--- a/src/app/event.service.ts
+++ b/src/app/event.service.ts
@@ -227,6 +227,7 @@ export class EventService {
   private applyCategories(monthyear: string) {
     console.log("APPLYING CATEGORIES");
     // apply monthyear here
+    console.log (moment(monthyear));
     let tempEvents = new FeatureCollection([]);
       for (let event of this._events.features) {
         let allSelected = this._categHash['all'].selected;
